examples/step-58/doc/intro.dox

```diff
@@ -0,0 +1,875 @@
+Geodynamics initiative (CIG), through the National Science Foundation under
+Award No. EAR-1550901 and The University of California-Davis.
+</i>
+
+<a name="Intro"></a>
+<h1>Introduction</h1>
+
+The <a
+href="https://en.wikipedia.org/wiki/Nonlinear_Schr%C3%B6dinger_equation">Nonlinear
+Schr&ouml;dinger Equation (NLSE)</a> for a function $\psi=\psi(\mathbf
+x,t)$ and a potential $V=V(\mathbf x)$ is a model often used in
+quantum mechanics and nonlinear optics. If one measures in appropriate
+quantities (so that $\hbar=1$), then it reads as follows:
+@f{align*}{
+  - i \frac{\partial \psi}{\partial t}
+  - \frac 12 \Delta \psi
+  + V \psi
+  + \kappa |\psi|^2 \psi
+  &= 0
+  \qquad\qquad
+  &
+  \text{in}\; \Omega\times (0,T),
+  \\
+  \psi(\mathbf x,0) &= \psi_0(\mathbf x)
+  &
+  \text{in}\; \Omega,
+  \\
+  \psi(\mathbf x,t) &= 0
+  &
+  \text{on}\; \partial\Omega\times (0,T).
+@f}
+If there is no potential, i.e. $V(\mathbf x)=0$, then it can be used
+to describe the propagation of light in optical fibers. If $V(\mathbf
+x)\neq 0$, the equation is also sometimes called the <a
+href="https://en.wikipedia.org/wiki/Gross%E2%80%93Pitaevskii_equation">Gross-Pitaevskii
+equation</a> and can be used to model the time dependent behavior of
+<a
+href="https://en.wikipedia.org/wiki/Bose%E2%80%93Einstein_condensate">Bose-Einstein
+condensates</a>.
+
+For this particular tutorial program, the physical interpretation of
+the equation is not of much concern to us. Rather, we want to use it
+as a model that allows us to explain two aspects:
+- It is a <b>complex-valued equation</b> for $\psi \in H^1(\Omega,{\mathbb
+  C})$. We have previously seen complex-valued equations in step-29,
+  but there have opted to split the equations into real and imaginary
+  parts and consequently ended up solving a system of two real-valued
+  equations. In contrast, the goal here is to show how to solve
+  problems in which we keep everything as complex numbers.
+- The equation is a nice model problem to explain how <b>operator
+  splitting methods</b> work. This is because it has terms with
+  fundamentally different character: on the one hand, $- \frac 12
+  \Delta \psi$ is a regular spatial operator in the way we have seen
+  many times before; on the other hand, $\kappa |\psi(\mathbf x,t)|^2
+  \psi$ has no spatial or temporal derivatives, i.e., it is a purely
+  local operator. It turns out that we have efficient methods for each
+  of these terms (in particular, we have analytic solutions for the
+  latter), and that we may be better off treating these terms
+  differently and separately. We will explain this in more detail
+  below.
+
+
+
+<h3>A note about the character of the equations</h3>
+
+At first glance, the equations appear to be parabolic and similar to
+the heat equation (see step-26) as there is only a single time
+derivative and two spatial derivatives. But this is misleading and
+more easily seen if we assume for a moment that the potential $V=0$
```

Comment:
>  But this is misleading and more easily seen

This reads a bit strangely to me. I can't help but interpret what is "more easily seen" as being the parabolic nature of the equations, which is clearly what you're trying to indicate is not correct. Maybe something like "But this is misleading and their true nature can be more easily seen" is better?